scripts/build-release-single-target

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source ./scripts/test_lib.sh
+
+VER=${1:-}
```

Comment:
Why not VERSION ?

Reply:
👍

Reply:
If we copied `scripts/build-binary`, this file should be named `scripts/build-binary-single-target`, otherwise we'll confuse it with `script/build-release.sh`